package.json

```diff
@@ -13,7 +13,7 @@
     "test:unit:silent": "npm run test:unit > tmp/test-unit-log.txt 2>&1",
     "test:browser": "karma start --single-run",
     "test:watch": "npm run test:unit -- --watch",
-    "coverage": "nyc --reporter=text --include='src/**/*.js' --temp-dir=./tmp/ --check-coverage --lines 50 npm run test:unit:silent",
+    "coverage": "nyc --reporter=text --include='src/**/*.js' --temp-dir=./tmp/ --check-coverage --lines 91 npm run test:unit:silent",
```

Comment:
nice 🙌 👏 👍 

not a blocker but FYI ideally we'd set our coverage threshold based on "branches" not merely "lines" but last i checked, `nyc` didn't support that flag 👉 https://github.com/istanbuljs/nyc#common-configuration-options